to.etc.domui/src/main/java/to/etc/domui/dom/html/Page.java

```diff
@@ -875,6 +876,14 @@ public void appendJS(@NonNull final CharSequence sq) {
 
 	@Nullable
 	public StringBuilder internalFlushAppendJS() {
+		if(internalCanLeaveCurrentPageByBrowser()) {
```

Comment:
These checks need to be the other way around. There is no need to do anything if that interface is not there.